backend/api/Controllers/EventController.cs

```diff
@@ -16,19 +16,13 @@ public EventController(ILogger<EventController> logger, EventService eventServic
         _logger = logger;
```

Comment:
The logger is no longer in use in the event controller. Should we log more, or is sufficient logging covered? If there is no need for logging in the event controller, can this be deleted?

Reply:
Fixed